.github/workflows/deploy_passdemo.yml

```diff
@@ -0,0 +1,33 @@
+name: Deploy passdemo
+
+on: workflow_dispatch
+
+jobs:
+  deploy_code:
+     runs-on: passdemo
+     steps:
+      - uses: actions/checkout@v2
+
+      - name: Code Sanity Check
+        run: |
+          echo "🎉 The job was automatically triggered by a ${{ github.event_name }} event."
+          echo "🐧 This job is now running on a ${{ runner.os }}"
+          pwd
+          ls -la
+
+      - name: Docker Sanity Check
+        run: |
+          groups
+          docker info
+
+      - name: Stop Existing App
+        run: |
+          docker-compose down
+
+      - name: Build App
+        run: |
+          docker-compose pull
+
+      # - name: Run App
```

Comment:
Turning off for now... will put back on when app confirmed to be safe to run on an open IP/URL